### DISCUSSION TITLE
Change the form data before submitting to the api for Create and Edit components

### DISCUSSION THREAD

**Discussion by Mikilll94** [refinedev/refine#3998] [General]
I have read this point from FAQ:
https://refine.dev/docs/faq/#how-can-i-change-the-form-data-before-submitting-it-to-the-api

<img width="872" alt="Screenshot 2023-03-28 at 12 14 31" src="https://user-images.githubusercontent.com/17653648/228204834-fe4c0a12-66b4-4319-9028-91697e2532c8.png">

How can I apply this to `Create` and `Edit` components? These components does not give access to `form` or `onSubmit` method?

**Comment by omeraplak** [comment]
Hey @Mikilll94 ,
As stated in the documents, Basic View does not contain any logic but adds extra functionalities like action buttons and giving titles to the page.
